.github/workflows/security.yaml

```diff
@@ -0,0 +1,19 @@
+on: push
+
+name: Security
+
+jobs:
+  ensure-pinned-actions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@f4a75cfd619ee5ce8d5b864b0d183aff3c69b55a # v2.13.1
```

Comment:
not a blocker, calling out for visibility - but we seem to be using 2.11.1 in other workflows.